### PR TITLE
chore(windows): help titles and missing files

### PR DESCRIPTION
**Pull request by mcdurdin**
This PR updates converts the initial heading line to a `title:` meta line and converts a few remaining files that I had missed from docbook to markdown. No content changes -- these are coming soon.

### CODE REVIEW

File: windows/src/desktop/help/basic/config_tasks/config_menu.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+title: Keyman Configuration {#basic_config_menu}
+---
+
+Keyman Configuration is the central control panel for Keyman. Use Keyman
+Configuration to purchase and activate Keyman, switch the language of
+Keyman menus, install and uninstall keyboards, change options, set
+hotkeys, disable addins, associate languages, check for updates and run
+support diagnostics.
+
+![](desktop_images/tab-support.png)
```
Comment (darcywong00) [inline windows/src/desktop/help/basic/config_tasks/config_menu.md:11]: Are the desktop_images up two folders? also for l. 17
`../../desktop_images/...png`

File: windows/src/desktop/help/basic/config_tasks/hotkeys_tab.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+To open the Hotkeys tab of Keyman Configuration:
+
+1.  Click on the Keyman icon ![](desktop_images/icon-keyman.png), on the
+    Windows Taskbar near the clock.
+
+2.  From the Keyman menu, select Configuration....
+
+3.  Select the Hotkeys tab.
+
+    ::: {.tip}
```
Comment (darcywong00) [inline windows/src/desktop/help/basic/config_tasks/hotkeys_tab.md:21]: What does `:::` do?

File: windows/src/desktop/help/basic/toolbox_tasks/charactermap.md [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+:::
+
+**Character Map Options**
+
+![](desktop_images/charmap-options.png)
+
+Right click on the Character Map to display a menu with the following
+options:
+
++-----------------+-----------------------------------------------------+
```
Comment (darcywong00) [inline windows/src/desktop/help/basic/toolbox_tasks/charactermap.md:55]: Do these tables need to be adjusted for Markdown?

File: windows/src/desktop/help/build.sh [hunk 4]
```diff
@@ -77,7 +77,7 @@ build_hhc_header() {
   echo '
 <HTML>
   <HEAD>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="Content-Type" content="text/html; charset=windows-1252" />
```
Comment (darcywong00) [inline windows/src/desktop/help/build.sh:80]: Why is this changing from utf-8 to windows-1252?

Comment (mcdurdin) [reply windows/src/desktop/help/build.sh:80]: The .hhc file is used by the HTML Help Workshop / HTMLHelp which does not have good support for Unicode in its index. Unfortunately, there are not any great replacements for the .chm file format at present, apart from going with fully web-based documentation (whether stored locally or on help.keyman.com).

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**